CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ This file is influenced by http://keepachangelog.com/.
 
 ### Fixed
 
+- Improved accessibility
```

Comment:
and making skip links work on all pages, and removing tabindex values > 1. Seemed easier not to be specific.